css/main.css

```diff
@@ -0,0 +1,431 @@
+  color: #fff;
+}
+
+.gtime {
+  background-image: url("../media/png/gtime.png");
+}
+
+.weather {
+  background-image: url("../media/png/weather.png");
+}
+
+.conference {
+  background-image: url("../media/png/conference.png");
+}
+
+.katchy {
+  background-image: url("../media/png/katchy.png");
+}
+
+.project-stacks {
+  display: flex;
+  flex-wrap: wrap;
+  grid-gap: 15px;
+  padding: 20px 0;
+}
+
+.project-stacks li {
+  display: inline;
+  border: 1px solid #8993a4;
+  padding: 10px 12px;
+  background: transparent;
+  color: #fff;
+  transition: 350ms;
+  cursor: pointer;
+}
+
+.project-stacks li:hover {
+  background: rgba(0, 0, 0, 0.349);
+}
+
+.about-me,
+.about-me-section {
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+}
+
+.about-me-text {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 24px;
+  color: #f4f5f7;
+}
+
+.link {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  padding: 12px;
+  background: #ff6b00;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  color: #fff;
+  align-self: start;
+  transition: 350ms;
+}
+
+.link:hover {
+  background: #a04300;
+}
+
+.about-me-icon {
+  position: absolute;
+  right: 0;
+  bottom: 0;
+}
+
+.list-title {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 32px;
+  line-height: 44px;
+}
+
+.languages {
+  list-style: inside url("../media/svg/rectangle.svg");
+}
+
+.frameworks {
+  list-style: inside url("../media/svg/square.svg");
+}
+
+.skills {
+  list-style: inside url("../media/svg/circle.svg");
+}
+
+.form-container {
+  width: 80%;
+  margin: 0 auto;
+  position: relative;
+  background-color: #fff;
+  padding-bottom: 100px;
+}
+
+.form {
+  position: relative;
+  width: 100%;
+  display: flex;
+  column-gap: 70px;
+  justify-content: center;
+  align-items: center;
+  margin-top: 50px;
+}
+
+.form-header {
+  width: 35%;
+  color: #262626;
+  font-size: 32px;
+  font-weight: 700;
+  line-height: 44px;
+  text-align: center;
+}
+
+.form-group {
+  position: relative;
+  width: 60%;
+  margin-top: 50px;
+  display: flex;
+  justify-content: space-around;
+  align-items: flex-start;
+  flex-direction: column;
+  row-gap: 25px;
+}
+
+.form-control {
+  position: relative;
+  width: 100%;
+}
+
+.form-control input {
+  width: 100%;
+  border: none;
+  outline: none;
+  border-bottom: 1px solid grey;
+  padding: 5px;
+}
+
+textarea {
+  width: 100%;
+  border: none;
+  outline: none;
+  background-color: #fbf8f7;
+  padding: 5px;
+}
+
+.form-button {
+  position: relative;
+  width: 131px;
+  height: 48px;
+  background-color: #ff6b00;
+  color: #fff;
+  padding: 12px;
+  border: none;
+  outline: none;
+  cursor: pointer;
+}
+
+.form-button:hover {
```

Comment:
Please, review that the button interaction works correctly when you hover over the button.

https://user-images.githubusercontent.com/58642949/183975328-1cab70f5-550b-4605-a477-3d7803e15a53.mp4

Reply:
Thank you for the review, Please I would like you to note that the hover effect can not be noticed on the mobile view (unless the button is clicked). Please check the desktop version to notice this effect. Thank you